test/protocol/OfferHandlerTest.js

```diff
@@ -2102,7 +2100,7 @@ describe("IBosonOfferHandler", function () {
           ).to.revertedWith(RevertReasons.INVALID_DISPUTE_RESOLVER);
         });
 
-        it("For some offer, dispute resolver is not active", async function () {
+        it.skip("For some offer, dispute resolver is not active", async function () {
```

Comment:
maybe add a `// TODO - revisit when account deactivations are supported` above the test.

Reply:
done